CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Calls to hashlib now go through helper functions
 ### Fixed
 - Record object key properties are now always strings
+- Download progressbar only reports min 1% changes, to avoid flooding logs
```

Comment:
```suggestion
```

Reply:
Ok , but would it be fine if CI for changelog i.e. `Tests / lint (changelog, 3.7, 12.x) ` would fail because of no changes in changelog  ??